modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/chart/bar/SimpleBarChart.es.js

```diff
@@ -41,6 +42,10 @@ export default ({data, height, totalEntries, width}) => {
 	};
 
 	const CustomizedYAxisTick = ({payload, x, y}) => {
+		const minX = -162;
+		const REMOVE_SPACES_REGEX = /\s/g;
```

Comment:
Hey @rebsilva, this regex actually looks for the white spaces only, therefore how about name it as `SPACES_REGEX` instead?

Reply:
Sure, i will change it!